### PR TITLE
:sparkles: add chat

### PR DESCRIPTION
**Pull request by larsdouweschuitema**
Signed-off-by: Lars Douwe Schuitema <info@larsschuitema.nl>

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+var socket = io();
+
+var btn = document.getElementById('button');
+var form = document.querySelector('form');
+var input = document.getElementById('m');
+var messages = document.getElementById('messages');
+
+form.addEventListener('submit', function(e) {
+
+	e.preventDefault();
```
Comment (rijkvanzanten) [inline src/js/main.js]: indentation? What happend to `2 spaces`

File: views/partials/footer.ejs [hunk 2]
```diff
@@ -1,5 +1,3 @@
 </main>
-<script src="/socket.io/socket.io.js"></script>
-<script>
-  var socket = io();
-</script>
+<script src="https://cdn.socket.io/socket.io-1.2.0.js"></script>
```
Comment (rijkvanzanten) [inline views/partials/footer.ejs]: Prefer bundled over `/socket.io/socket.io.js`.. Prefer `/socket.io/socket.io.js` over `https://cdn.socket.io/socket.io-1.2.0.js`

Comment (larsdouweschuitema) [inline views/partials/footer.ejs]: Issue #4

**Comment by larsdouweschuitema** [comment]
@rijkvanzanten I updated the indentation

File: src/js/main.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  e.preventDefault();
+  socket.emit('chat message', input.value);
+  return false;
+});
+
+socket.on('chat message', function (msg) {
+  var message = document.createElement('li');
+  message.innerHTML = msg;
+  messages.appendChild(message);
+});
```
Comment (wooorm) [inline src/js/main.js]: Missing newline at end of file!

Comment (rijkvanzanten) [inline src/js/main.js]: Blasphemy!

**Pull request merged**